php/libraries/Utility.class.inc

```diff
@@ -198,7 +198,7 @@ class Utility
         if ($study_site) {
             $query .= "WHERE Study_site='Y'";
         }
-
+        $query .= "ORDER BY Name";
```

Comment:
This goes into an array, so the order returned by SQL shouldn't be relevant.. the PHP documentation (http://php.net/manual/en/language.types.array.php) specifies that arrays are ordered, but doesn't specify what the ordering is (It's only seems to be implied that it's the order that the keys are set.)

It does, however, have a number of sorting functions (http://php.net/manual/en/array.sorting.php).. I think it would be safer to explicitly sort from PHP using natsort or natcasesort after building the array and include a comment specifying that it preserves keys and linking to the PHP array sorting documentation.